server/src/config.js

```diff
@@ -13,10 +13,17 @@ let conf = convict({
     env: "PORT",
     arg: "port"
   },
+  host: {
+    doc: "The server host",
+    format: String,
+    default: "localhost",
+    env: "HOST",
+    arg: "host"
+  },
   contentPort: {
```

Comment:
Are we expecting to run two ports in production?  We could also do Host-header matching in that case.  Mostly because it makes the nginx configuration a bit easier, which isn't actually that challenging (except with ports it means nginx has to have the same domain mappings that the configuration will have).